sdk/resourcemanager/Azure.ResourceManager.Core/src/ApiVersions.cs

```diff
@@ -20,7 +18,7 @@ namespace Azure.ResourceManager.Core
     /// </summary>
     public class ApiVersions
     {
-        private ProvidersOperations ProviderOperations;
+        private ProviderRestOperations ProviderRestOperations;
```

Comment:
given this is a "consumer" of the providers operations I think this should be a providersoperations type.  The only thing that should access the providersrestoperation is the providersoperation and providerscontainer classes